CNAME

```diff
@@ -1 +0,0 @@
-about.workflowhub.eu
```

Comment:
We still need this for the domain to work, right @stain ?

Reply:
Yes, the `CNAME` file must have been removed for @bedroesb to preview - but it needs to not be removed by this PR!